api/V1/Device.js

```diff
@@ -279,4 +279,51 @@ module.exports = function(app, baseUrl) {
       });
     })
   );
+
+  /**
+   * @api {post} /api/v1/devices/query Querys devices by group:name.
+   * @apiName query
+   * @apiGroup Device
+   * @apiDescription This call is to query all devices by groups or devicenames
+   *
+   * @apiUse V1DeviceAuthorizationHeader
+   *
+   * @apiParam {JSON} devices array of device obejcts [{devicename:"<name>",groupname:"<group>"}].
+   * @apiParam {String} groups array of group names.
+   * @apiParam {String} operator sequelize operator to join devices and groups.
+   * by defualt $or "device in devices $OR device in groups"
```

Comment:
default

Reply:
please, no more sequelize leaking out of the API! I'm fine with a parameter for specifying this behaviour but we shouldn't be feeding input from the API straight into the ORM layer.

I know we do it elsewhere but I want to gradually get rid of those cases. Let's not add more.